housing/housing.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+from lxml import html
+import requests
+import time
+
+
+HEADERS = {'User-Agent':
+           'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0'}
+
+
+def scrap_housing():
+    place = raw_input('Enter city: ').lower()
```

Comment:
if this script is to be called by other scripts, like the crawler, then the arguments need to be command line